src/kbmod/search/stamp_creator.cpp

```diff
@@ -68,7 +68,8 @@ std::vector<RawImage> StampCreator::get_coadded_stamps(ImageStack& stack, std::v
     rs_logger->info("Performing stamp filtering on " + std::to_string(t_array.size()) + " trajectories.");
     DebugTimer timer = DebugTimer("stamp filtering", rs_logger);
 
-    if (use_gpu) {
+    // We use the GPU if we have it for everything except STAMP_VAR_WEIGHTED which is CPU only.
```

Comment:
is being CPU only a limit of the formula for the weighted var stamps or are we just holding off on the gpu implementation for now? I'm fine either way :)

Reply:
It's a "holding off for now". The GPU version is a bit more tricky because, unlike the other stamp functions, we will need to transfer the variance layers to the GPU as well. Instead of doing all that machinery now, I wanted to put something in place that we could test. If it is useful and a bottleneck, we can optimize on the GPU.